src/NamingStrategy/PathNamingStrategy.php

```diff
@@ -0,0 +1,85 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Http\Client\Plugin\Vcr\NamingStrategy;
+
+use Psr\Http\Message\RequestInterface;
+use Symfony\Component\OptionsResolver\Options;
+use Symfony\Component\OptionsResolver\OptionsResolver;
+
+/**
+ * Will use the request path as filename.
+ *
+ * @author Gary PEGEOT <garypegeot@gmail.com>
+ */
+class PathNamingStrategy implements NamingStrategyInterface
+{
+    /**
+     * @var array
+     */
+    private $options;
+
+    public function __construct(array $options = [])
```

Comment:
please add phpdoc for the options so that the user is aware of what options can be passed. notably the hash_body_methods are not clear until reading the actual code.

Reply:
I documented the new options, and removed hostname and suffix

Reply:
thanks!